src/App.css

```diff
@@ -68,11 +68,26 @@ p {
   justify-content: flex-start;
   padding: 38px;
   border-bottom: 0.7px solid #fff;
+  position: relative; /* Added for positioning arrows */
 }
 
+/* New styles for category name and value */
 .coin h3,
 p {
   margin: 0;
+  position: absolute;
+  bottom: 10px;
+  right: 28px;
+  text-align: right;
+  padding: 10px;
+}
+
+/* New style for right arrow */
+.arrow {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  font-size: 1.2rem;
 }
 
 .lastcoin {
```

Comment:
- [ ] Please add right arrows aligned at the top right corner of each category container, as shown in the design guidelines below. 
- [ ] Adjust the style to prevent overlap between the category name and price, caused by the changes made to align them at the right corner.
  
  ### Your design
  ![image](https://github.com/maths29/metricswebapp/assets/92303891/281b3c98-978f-461b-9ff0-e769ca322143)
  
  ### Design guidelines
  ![image](https://github.com/maths29/metricswebapp/assets/92303891/0e16dca9-1258-4868-a6a7-b6a5bf3dbc70)